packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.controller.js

```diff
@@ -0,0 +1,137 @@
+import { DNS_FILENAME, DATE_FORMAT } from './dashboard.constants';
+
+export default class DomainDnsZoneHistoryDashboardController {
+  constructor(
+    $translate,
+    $stateParams,
+    $state,
+    $q,
+    $document,
+    Domain,
+    Alerter,
+    DNSZoneService,
+    goBack,
+    goToDiffViewer,
```

Comment:
I guess you dont need to pass this to constructor.

Reply:
Yes indeed, only in case I changed imports to component instead separated (ctlers, template) => I did the change